File: python/7 kyu/180.py
# https://www.codewars.com/kata/580a4734d6df748060000045/train/python

# Passed

def is_sorted_and_how(arr):
    sorted_arr = list(sorted(arr))
    if arr == sorted_arr:
        return "yes, ascending"
    elif arr == sorted_arr[::-1]:
        return "yes, descending"
    else:
        return "no"
    
output = is_sorted_and_how([15, 7, 3, -8])
print(output)

File: nodejs/7 kyu/180.js
// https://www.codewars.com/kata/58a369fa5b3daf464200006c/train/javascript

// Passed

function getPercentage(sent, limit) {
  limit = limit ? limit : 1000;

  if (sent === 0) {
    return "No e-mails sent";
  } else if (sent >= limit) {
    return "Daily limit is reached";
  } else {
    let percent = Math.floor((sent / limit) * 100);
    let result = `${percent}%`;
    return result;
  }
}

const output = getPercentage(256, 500);
console.log(output);